pkg/kubectl/cmd/annotate.go

```diff
@@ -107,7 +109,7 @@ func NewCmdAnnotate(f cmdutil.Factory, out io.Writer) *cobra.Command {
 
 	cmd := &cobra.Command{
 		Use:     "annotate [--overwrite] (-f FILENAME | TYPE NAME) KEY_1=VAL_1 ... KEY_N=VAL_N [--resource-version=version]",
-		Short:   "Update the annotations on a resource",
+		Short:   T("annotate_short"),
```

Comment:
If we do this, I think the pattern should be constants for all translated string keys.  One constant file per resource bundle with a static analyzer to catch mismatches.  I've had better luck with that.